src/depackers/stb_vorbis.c

```diff
@@ -2135,33 +2380,27 @@ void inverse_mdct_slow(float *buffer, int n, vorb *f, int blocktype)
    }
    free(x);
 }
-#else
+#elif 0
 // transform to use a slow dct-iv; this is STILL basically trivial,
 // but only requires half as many ops
-#if 0
-static void dct_iv_slow(float *buffer, int n)
```

Comment:
All `dct_iv_slow` and `inverse_mdct_slow` should be static

Reply:
This is all debug deadcode and is irrelevant for libxmp IMO.